src/store/calendarObjectInstance.js

```diff
@@ -1424,7 +1438,12 @@ const actions = {
 		const calendarObject = await dispatch('createNewEvent', { start, end, isAllDay, timezoneId })
 		const startDate = new Date(start * 1000)
 		const eventComponent = getObjectAtRecurrenceId(calendarObject, startDate)
-		const calendarObjectInstance = mapEventComponentToEventObject(eventComponent)
+		const eventProps = objectFromEntries(Object.entries({ title, description }).filter(([k, v]) => v != null))
+		const calendarObjectInstance = Object.assign(
```

Comment:
I didn't yet have a chance to test this, but I doubt this will work.

You are adding the properties to the `calendarObjectInstance`, but not to the actual `eventComponent`. So unless you are editing the title, no title will be set when hitting the save button.

Reply:
Thanks. Fixed. I was assuming that if there is something in input and I hit save, it's gonna be saved - didn't test it.